src/apps/vpn/ui/main.qml

```diff
@@ -14,6 +14,10 @@ import components 0.1
 import org.mozilla.Glean 0.30
 import telemetry 0.30
 
+
```

Comment:
Note: this won't land, i just want to make sure the linter runs only against files touched in a pr :)